src/main/kotlin/net/atos/zac/app/task/converter/RestTaskConverter.kt

```diff
@@ -118,7 +118,7 @@ class RestTaskConverter @Inject constructor(
     }
 
     private fun readFormioFormulier(formulierNaam: String): JsonObject {
-        return this.javaClass.getResourceAsStream("formulieren/%s.json".format(formulierNaam))!!
+        return this::class.java.classLoader.getResourceAsStream("formulieren/%s.json".format(formulierNaam))!!
```

Comment:
Strange that `this.javaClass` does not work.. How can we test this?

Reply:
Maybe you can give input in https://dimpact.atlassian.net/browse/PZ-3857 as to how we can most easily test this BPMN PoC functionality?